ecosystem/notes/YYYY-MM-DD-ecosystem-call.md

```diff
@@ -0,0 +1 @@
+This is a placeholder for the first ecosystem call. Note that the naming convention for all notes should follow the `YYYY-MM-DD-ecosystem-call.md` convention.
```

Comment:
Can I be a bit of a nitpick and suggest we go with the format that's already in `/notes`, i.e. 
`YYYYMMDD-ecosystem-call.md`?